src/extension/src/file_handlers/ExtOutputStatusHandler.py

```diff
@@ -62,17 +59,18 @@ def __init__(self, logger, utility, json_file_handler, log_file_path, seq_no, di
         self.__current_operation = None
 
         # Load the currently persisted status file into memory
-        self.read_file()
-
-    def write_status_file(self, operation, status=Constants.Status.Transitioning.lower()):
-        self.logger.log("Writing status file to provide patch management data for [Sequence={0}]".format(str(self.__seq_no)))
+        #ToDo: move it to some other location, since seq no is not available at load
+        # self.read_file()
```

Comment:
This is a bit worrisome. Please document the design for verification including all expected state transitions and the expected behavior of the extension in all those transitions (including benign re-enables, legitmate reboots, etc.). The code can then be double-checked against that.

Reply:
To speed up each review, please keep individual review complexity low - writing to the status for non-enable operations is not integral to this review. (As in the additional ask to split out log grooming between ext. and core, to avoid loss of important logs when benign 'enables' overflow)

Reply:
Will include this with the next PR on writing to status file